addons/l10n_latam_invoice_document/wizards/account_move_reversal.py

```diff
@@ -32,10 +32,17 @@ def _reverse_type_map(self, move_type):
             'in_receipt': 'out_receipt'}
         return match.get(move_type)
 
-    @api.depends('move_ids')
+    @api.depends('l10n_latam_available_document_type_ids')
     def _compute_document_type(self):
+        for record in self.filtered(
+                lambda x: not x.l10n_latam_document_type_id or
+                x.l10n_latam_document_type_id not in x.l10n_latam_available_document_type_ids):
+            document_types = record.l10n_latam_available_document_type_ids._origin
```

Comment:
@jjscarafia Why do you do ._origin here?

Reply:
@zaoral if time allows please check this

Reply:
@jjscarafia @jco-odoo @zaoral @vbe-odoo 
I think the reason is that the "_origin" was already on the original method.. so we should check if we can remove it from there too.. this is the [line](https://github.com/odoo/odoo/blob/19a2f84b39779f7bb68804246bfe28d106da812d/addons/l10n_latam_invoice_document/wizards/account_move_reversal.py#L46)

Reply:
Without the ._origin it doesn't work properly. I need to added to make it work. If you remove it and play with it you will find the error (if not, I can make it on my return).

I remember wi discussed the original "._origin" and was something related to a limitartion / bug of the ORM that on m2m computed fields was not working as other fields (._origin solve it)

@Bruno-Zanotti @zaoral @jco-odoo

Reply:
@jco-odoo could you check this last comment from @jjscarafia about the "._origin", thanks!

Reply:
@jjscarafia if you can share the use case (example or screenshots) w/the issue will be nice to have it